capsules/src/process_console.rs

```diff
@@ -221,7 +221,7 @@ impl<U: UART, C: ProcessManagementCapability> ProcessConsole<'a, U, C> {
                                 );
                             });
                         } else if clean_str.starts_with("list") {
-                            debug!(" PID    Name                Slices  Syscalls  Dropped Callbacks    State");
+                            debug!(" PID    Name               Quantas  Syscalls  Dropped Callbacks    State");
```

Comment:
The plural of quantum is quanta. Quantas is an airline. :)

Reply:
Sigh..

New Tock sponsor?! I'd leave it if they chipped in for some paid engineers :)